src/constants/constants.js

```diff
@@ -1,6 +1,6 @@
 import { words } from '../data/data';
 
-const WORD_LENGTH = 7;
-const WORDS_COUNT = words.length;
-
-export { WORD_LENGTH, WORDS_COUNT };
+export const WORD_LENGTH = 7;
+export const WORDS_COUNT = words.length;
+export const DURATION = 120000;
+export const INTERVAL = 1000;
```

Comment:
I would say typically the name of such folder+file is `config`. It's good as it is, just sharing what came to my mind! :)

Reply:
👍